packages/ui-scripts/lib/fix-publish.js

```diff
@@ -40,13 +40,20 @@ const { createNPMRCFile } = require('./utils/npm')
 
 try {
   const pkgJSON = getPackageJSON()
-  // optional version argument
-  // ui-scripts --fix-publish 5.12.2
+  // Arguments
+  // 1: version to publish. If current version, use 'current' otherwise e.g.: 8.1.3
+  // 2: publish type. defaults to current. If set to 'maintenance', it will publish with vx_maintenance tag
+  // e.g.: ui-scripts --fix-publish 5.12.2 maintenance
+  const releaseVersion =
+    process.argv[3] === 'current' || !process.argv[3]
+      ? pkgJSON.version
+      : process.argv[3]
   fixPublish(
     pkgJSON.name,
     pkgJSON.version,
-    process.argv[3] || pkgJSON.version,
-    getConfig(pkgJSON)
+    releaseVersion,
+    getConfig(pkgJSON),
+    process.argv[4]
```

Comment:
I think it would be more descriptive if we would change the signature of the `fixPublish` function to take a single object parameter instead of 4 parameters like we did it with the `publish` function.